diffulex/utils/loader.py

```
@@ -1,4 +1,4 @@
-import os
+﻿import os
```

Ruff (0.15.9)

[warning] The file is executable but no shebang is present

(EXE002)